### PR TITLE
Add virtio packed test cases in disk and scsi controller

### PR DESCRIPTION
**Pull request by meinaLi**
1. Add packed attribute test in virtio disk driver
2. Add packed attribute test in virtio-scsi controller

Signed-off-by: Meina Li <meili@redhat.com>

### CODE REVIEW

**Comment by meinaLi** [comment]
Test Result:
```
# avocado run --vt-type libvirt --vt-machine-type q35 virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk
JOB ID     : caf69c06ff39d1d0f54be82051f6ee2120f79c2f
JOB LOG    : /root/avocado/job-results/job-2020-12-02T04.07-caf69c0/job.log
 (1/4) type_specific.io-github-autotest-libvirt.virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk.lvm_disk_attach: PASS (62.81 s)
 (2/4) type_specific.io-gith# avocado run --vt-type libvirt --vt-machine-type q35 virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk
JOB ID     : caf69c06ff39d1d0f54be82051f6ee2120f79c2f
JOB LOG    : /root/avocado/job-results/job-2020-12-02T04.07-caf69c0/job.log
 (1/4) type_specific.io-github-autotest-libvirt.virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk.lvm_disk_attach: PASS (62.81 s)
 (2/4) type_specific.io-github-autotest-libvirt.virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk.with_packed.packed_on: PASS (71.92 s)
 (3/4) type_specific.io-github-autotest-libvirt.virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk.with_packed.packed_off: PASS (71.95 s)
 (4/4) type_specific.io-github-autotest-libvirt.virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk.with_packed.in_scsi_controller: PASS (72.61 s)
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 285.86 s
ub-autotest-libvirt.virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk.with_packed.packed_on: PASS (71.92 s)
 (3/4) type_specific.io-github-autotest-libvirt.virtual_disks.at_dt_iscsi_disk.positive_test.c# avocado run --vt-type libvirt --vt-machine-type q35 virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk
JOB ID     : caf69c06ff39d1d0f54be82051f6ee2120f79c2f
JOB LOG    : /root/avocado/job-results/job-2020-12-02T04.07-caf69c0/job.log
 (1/4) type_specific.io-github-autotest-libvirt.virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk.lvm_disk_attach: PASS (62.81 s)
 (2/4) type_specific.io-github-autotest-libvirt.virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk.with_packed.packed_on: PASS (71.92 s)
 (3/4) type_specific.io-github-autotest-libvirt.virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk.with_packed.packed_off: PASS (71.95 s)
 (4/4) type_specific.io-github-autotest-libvirt.virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk.with_packed.in_scsi_controller: PASS (72.61 s)
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 285.86 s
old_plug.block_disk.with_packed.packed_off: PASS (71.95 s)
 (4/4) type_specific.io-github-autotest-libvirt.virtual_disks.at_dt_iscsi_disk.positive_test.cold_plug.block_disk.with_packed.in_scsi_controller: PASS (72.61 s)
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 285.86 s
```

**Comment by meinaLi** [comment]
It depend on https://github.com/avocado-framework/avocado-vt/pull/2843

File: libvirt/tests/src/virtual_disks/at_dt_iscsi_disk.py [hunk 1]
```diff
@@ -93,6 +98,10 @@ def run(test, params, env):
         if not libvirt_version.version_compare(4, 7, 0):
             test.cancel("iscsi-direct pool is not supported in"
                         " current libvirt version.")
+    if disk_packed or scsi_packed:
```
Comment (chunfuwen) [inline libvirt/tests/src/virtual_disks/at_dt_iscsi_disk.py:101]: if scsi_packed is true, that means that disk_packed should be true , so 'if disk_packed ' is enough as condition

Comment (meinaLi) [reply libvirt/tests/src/virtual_disks/at_dt_iscsi_disk.py]: Updated

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**